pages/api/users/index.js

```diff
@@ -23,7 +26,12 @@ export default async function handler(req, res) {
     return fileProfiles;
   }
 
-  const profiles = fileProfiles.map((fileProfile) => {
+  if(initialRandomUsers) {
```

Comment:
Please use the Prettier plugin on VScode, your coding style does not match the projects